repositories/AnswerRepository.js

```diff
@@ -1,21 +1,24 @@
 const { head, invert, map } = require("lodash/fp");
 const Answer = require("../db/Answer");
 const mapFields = require("../utils/mapFields");
-
 const mapping = { QuestionPageId: "questionPageId" };
 const fromDb = mapFields(mapping);
 const toDb = mapFields(invert(mapping));
 
 module.exports.findAll = function findAll(
   where,
+  whereNot = { isDeleted: true },
   orderBy = "created_at",
   direction = "asc"
 ) {
-  return Answer.findAll(where).orderBy(orderBy, direction).then(map(fromDb));
+  return Answer.findAll(where)
+    .whereNot(whereNot)
+    .orderBy(orderBy, direction)
+    .then(map(fromDb));
 };
 
-module.exports.get = function get(id) {
-  return Answer.findById(id).then(fromDb);
+module.exports.get = function get(id, where = { isDeleted: false }) {
```

Comment:
I wonder whether it's even worth having this `where` param for `get`? You're trying to select a single item, so it doesn't make sense to select on anything other than `id` (at least from the perspective of the calling code)